tests/examples/test_examples.py

```diff
@@ -47,15 +61,17 @@ def get_python_files() -> list[Path]:
     return find_python_files(examples_dir)
 
 
-def get_use_cases_dirs() -> list[Path]:
+def get_use_cases_dirs(
+    ignore_list: tuple[str, ...] = ("__pycache__", "github-auto-issues-trending-repos", "email-notifier-agent"),
```

Comment:
is there a way to still run `email-notifier-agent` ? We can setup the stmp variables right ?